TheWarWithin/RogueAssassination.lua

```diff
@@ -2739,6 +2586,7 @@ spec:RegisterAbilities( {
             gain( action.shiv.cp_gain, "combo_points" )
             removeDebuff( "target", "dispellable_enrage" )
             if talent.improved_shiv.enabled then applyDebuff( "target", "shiv" ) end
+            if talent.supercharger.enabled then addStack( "supercharged", nil, talent.supercharger.rank ) end
```

Comment:
Like Echoing Reprimand, if we can't know which combo point is Supercharged, we don't want to fake it.